### ISSUE TITLE
Use text and the required attribute to indicate required fields

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#168]
[atoms/atoms.input](https://a-ui.github.io/core_branding_scss/#atoms-input)

The following recommendations are accessibility improvements but not required to achieve WCAG compliance.

In the current implementation, required fields are marked with an asterisk that is inserted with CSS.

```html
<div class="a-input is-required">
  <label class="a-input__label" for="required-text-field">Required text field</label>
  <input type="text" name="required-text-field" id="required-text-field" placeholder="Placeholder" />
</div>
```

```css
.a-input.is-required .a-input__label:after {
    color: #da291c;
    content: ' *';
}
```

* Consider using text "required" instead of an asterisk. It's more user friendly and accessible.
* If you do use an asterisk, always provide an explanation above the form. For example: Fields marked with an asterisk (*) are required. Add an a11y-note?
* Add the HTML `required` attribute to the input to make screenreaders announce it as a required field. 

### Links and articles

* [Nielsen Norman group: Marking Required Fields in Forms](https://www.nngroup.com/articles/required-fields/)
* [Deque: Clearly Identify Required Form Fields](https://dequeuniversity.com/tips/identify-required-elements)

**Comment by TriangleJuice** [comment]
Hey @Veyfeyken this issue should be fixed in the next iteration of our core branding (`v4.3.0`)
